.github/workflows/help_exit.yml

```diff
@@ -0,0 +1,75 @@
+name: Help-Exit
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'eask'
+      - '**.yml'
+      - lisp/**
+      - cmds/**
+      - src/**
+      - test/**
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - '**.md'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        emacs-version:
+          - 26.3
+          - 27.2
+          - 28.2
+          - 29.4
+          - 30.1
```

Comment:
I would prefer to test only the latest version since this is unlikely to break with different Emacs versions. WDYT? :)

Reply:
Agree. And just ubuntu as well?

Reply:
> And just ubuntu as well?

I'll test it on all operating systems since it's likely something that could easily break.

For reference: 

https://github.com/emacs-eask/cli/blob/999ce9328360f8eede1e2b6e8f5ba5ccfe9e1f13/.github/workflows/search.yml#L31-L33

Reply:
Ok changed back to test all OS'